javascript/index.js

```diff
@@ -1,65 +1,138 @@
 const chronometer = new Chronometer();
 
 // get the buttons:
-const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+const btnLeft = document.getElementById("btnLeft");
+const btnRight = document.getElementById("btnRight");
 
 // get the DOM elements that will serve us to display the time:
-let minDec = document.getElementById('minDec');
-let minUni = document.getElementById('minUni');
-let secDec = document.getElementById('secDec');
-let secUni = document.getElementById('secUni');
-let milDec = document.getElementById('milDec');
-let milUni = document.getElementById('milUni');
-let splits = document.getElementById('splits');
+let minDec = document.getElementById("minDec");
+let minUni = document.getElementById("minUni");
+let secDec = document.getElementById("secDec");
+let secUni = document.getElementById("secUni");
+let milDec = document.getElementById("milDec");
+let milUni = document.getElementById("milUni");
+let splits = document.getElementById("splits");
 
 function printTime() {
-  // ... your code goes here
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  let min = chronometer.getMinutes();
+  if (min > 9) {
+    let minString = String(min).split("");
+    console.log(minString);
+    minUni.innerText = minString[1];
+    minDec.innerText = minString[0];
+  } else {
+    minUni.innerText = min;
+    minDec.innerText = 0;
+  }
 }
 
 function printSeconds() {
-  // ... your code goes here
+  let sec = chronometer.getSeconds();
+
+  if (sec > 9) {
+    let secString = String(sec).split("");
+    console.log(secString);
+    secUni.innerText = secString[1];
+    secDec.innerText = secString[0];
+  } else if (sec == 0) {
+    secUni.innerText = sec;
+    secDec.innerText = 0;
+  } else {
+    secUni.innerText = sec;
+  }
```

Comment:
this code works good but could be simplified, mostly by using the twoDigitsNumber method.